tests/x11/chrome.pm

```diff
@@ -3,19 +3,20 @@ use testapi;
 
 sub run() {
     my $self = shift;
+    my $arch = "";
```

Comment:
leave out the = ""

you have an if and an else, so afterwards the variable is set anyway